### PR TITLE
Fixed issue #880

### PR DESCRIPTION
**Pull request by Satyam26**
Changed twitter logo to fossasia logo and added background color of navigation header

### CODE REVIEW

File: android/app/src/main/res/layout/nav_header.xml [hunk 1]
```diff
@@ -5,13 +5,16 @@
...
     <ImageView
         android:id="@+id/headerDrawer"
         android:layout_width="fill_parent"
         android:layout_height="fill_parent"
         android:contentDescription="@string/header_image"
-        android:src="@drawable/twitter" />
+        android:src="@mipmap/ic_launcher" />
 
 </LinearLayout>
+
```
Comment (mananwason) [inline android/app/src/main/res/layout/nav_header.xml]: Please remove extra lines.

Comment (Satyam26) [inline android/app/src/main/res/layout/nav_header.xml]: @mananwason sorry for the mistake ,actually i am beginner so i didn't know about that.

Comment (Satyam26) [inline android/app/src/main/res/layout/nav_header.xml]: @mananwason should i close this pr and generate a fresh one.

**Comment by Satyam26** [comment]
@mananwason sir as you told me to squash commits before pr and remove extra lines.I have removed extra lines but i am unable to squash the commits.When i am typing command "git rebase ..." this comes up :

Cannot rebase: You have unstaged changes.
Please commit or stash them.

when i do "git diff "and "git status" it shows two files created by you that are modified:
parsingExtras/Microlocation.java
parsingExtras/Track.java

i have tried a lot like add and commit or stashing but unable to execute rebase command for squashing ,i am trying but could not pass through so please help me out

**Comment by mananwason** [comment]
You have some changes that you have not committed yet. This is why you're getting the exception. You need to check what changes have been done by you. git status should return no changes when you have successfully committed everything. Let me know if you face any issues.

**Comment by Satyam26** [comment]
Thanks @mananwason

**Pull request closed without merging**